WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1133,11 +1134,31 @@ private extension ProductFormViewController {
         let blazeViewModel = BlazeWebViewModel(siteID: viewModel.productModel.siteID,
                                                source: source,
                                                siteURL: siteUrl,
-                                               productID: product.productID)
+                                               productID: product.productID) {
+            self.handlePostCreation()
+        }
```

Comment:
⛔️ Please keep a weak reference to `self` to avoid retain cycle.

Reply:
Thanks, added in https://github.com/woocommerce/woocommerce-ios/pull/11552/commits/5c526cccc058bf709f58be218c670d86ba9bf625